test.properties

```diff
@@ -112,6 +112,8 @@
     #browser.type=safari
 
     browser.type[tck]=chrome
+    browser.type[firefox]=firefox
+    browser.type[functional-smoke-firefox-jdk8]=firefox
```

Comment:
We have any logic currently that will parse this. The tck property is different, because that property is literally read in from the Jenkins side.